test/jobs/data_exports/create_job_test.rb

```diff
@@ -82,6 +83,46 @@ def setup
       assert_equal expected_manifest.to_json, @data_export2.manifest
     end
 
+    test 'content of sample export simple manifest' do
+      sample = samples(:sample1)
+      project = projects(:project1)
+      attachment1 = attachments(:attachment1)
+      attachment2 = attachments(:attachment2)
+      expected_files_in_zip = ["#{project.puid}/#{sample.puid}/#{attachment1.puid}/#{attachment1.file.filename}",
+                               "#{project.puid}/#{sample.puid}/#{attachment2.puid}/#{attachment2.file.filename}",
+                               'manifest.json',
+                               'manifest.txt']
+      actual_simple_manifest = ''
+
+      DataExports::CreateJob.perform_now(@data_export2)
+      export_file = ActiveStorage::Blob.service.path_for(@data_export2.file.key)
+      Zip::File.open(export_file) do |zip_file|
+        zip_file.each do |entry|
+          assert expected_files_in_zip.include?(entry.to_s)
+          actual_simple_manifest = entry.get_input_stream.read.force_encoding('utf-8') if entry.to_s == 'manifest.txt'
+          expected_files_in_zip.delete(entry.to_s)
+        end
+      end
+      assert expected_files_in_zip.empty?
+
+      expected_simple_manifest = [
+        'Samples Export',
+        '2024-08-14',
```

Comment:
Didn't notice until a failed test just now, this should be like other manifest tests `Date.current.strftime('%Y-%m-%d')`

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/700/commits/85c522dfbbcf65cd1f84bddf669f0413d6a990e5